third_party/intel/lib/TritonIntelGPUTransforms/MatchTargetSize.cpp

```diff
@@ -104,21 +104,15 @@ class MatchTargetSizePass
     MLIRContext *ctx = &getContext();
     ModuleOp m = getOperation();
 
-    // Collect `tt.dot` operations layouts.
+    // Collect the result layout of "interesting" `tt.dot` operations.
+    // A candidate 'tt.dot' operation yields a tensor(or pointer to tensor) with
```

Comment:
```suggestion
    // A candidate 'tt.dot' operation yields a tensor with
```
for dot, it can only be tensor